docs/api/admin-capella.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022-Present Couchbase, Inc.
+#
+# Use of this software is governed by the Business Source License included
+# in the file licenses/BSL-Couchbase.txt.  As of the Change Date specified
+# in that file, in accordance with the Business Source License, use of this
+# software will be governed by the Apache License, Version 2.0, included in
+# the file licenses/APL2.txt.
+
+openapi: 3.0.3
+info:
+  title: Sync Gateway
+  description: Sync Gateway manages access and synchronization between Couchbase Lite and Couchbase Server
+  version: 3.3.0
+  license:
+    name: Business Source License 1.1 (BSL)
+    url: 'https://github.com/couchbase/sync_gateway/blob/master/LICENSE'
+servers:
+  - url: 'https://{hostname}:4985'
+    description: Admin API
+    variables:
+      hostname:
+        description: The hostname to use
+        default: localhost
+paths:
+  '/{db}/_session':
+    $ref: './paths/admin/db-_session.yaml'
+    x-capella: true
```

Comment:
How come this has been added here but nowhere else?

Reply:
I thought this would work to use `filter-in` but it does not because filter-in only works if a property is defined. I think it'd be too much to remember to define a property on every operation.

This was an artifact from a previous experiment. Any properties with `x-` are ignored by openapi and redocly unless redocly gives them special meanings.